examples/mqtt-paho/main.c

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (C) 2019
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       MQTT-paho example
+ *
+ * @author      Javier FILEIV <javier.fileiv@gmail.com>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <thread.h>
+#include <mqtt.h>
+#include <log.h>
+#include "msg.h"
+#include "lwip.h"
+#include "lwip/netif.h"
+#include "net/ipv6/addr.h"
+#include "net/ipv4/addr.h"
+#include "MQTTClient.h"
+#include "shell.h"
+
+#define REMOTE_IP_ADDRESS       "fe80::18cf:f9ff:fe1c:afd2"
+#define MAX_MSGS                12
+#define BUF_SIZE                250
+
+static char* topic = "test_from_riot";
+MQTTClient client;
+static char _sender_stack[THREAD_STACKSIZE_DEFAULT];
+
+static void _messageArrived(MessageData* data)
```

Comment:
Maybe rename this function to:
```suggestion
static void  _on_msg_received(MessageData* data)
```
?